controllers/eventbus/installer/nats.go

```diff
@@ -436,7 +436,7 @@ func (i *natsInstaller) buildConfigMap() (*corev1.ConfigMap, error) {
 	ssName := generateStatefulSetName(i.eventBus)
 	replicas := i.eventBus.Spec.NATS.Native.GetReplicas()
 	if replicas < 3 {
-		replicas = 3
+		i.logger.Errorw("streaming recommends a replica size >= 3, '%v'", replicas)
```

Comment:
Shall we keep the legacy nats streaming untouched, since it's already EOL?

Reply:
I hadn't even thought about that,  do you have a preference?  Not sure how much NATS streaming is still used

Reply:
This is the file for nats streaming installation, I would recommend to keep it untouched.